genai-perf/genai_perf/llm_inputs/llm_inputs.py

```diff
@@ -591,7 +593,12 @@ def _add_rows_to_generic_json(
         return generic_input_json
 
     @classmethod
-    def _get_input_dataset_from_file(cls, input_filename: Path) -> Dict:
+    def _get_input_dataset_from_file(
+        cls,
+        input_filename: Path,
+        batch_size: int = DEFAULT_BATCH_SIZE,
+        num_prompts: int = -1,
```

Comment:
what does -1 represent here?

Reply:
An invalid default value so that any future failures are not silent.